src/com/trollworks/gcs/app/GCS.java

```diff
@@ -233,6 +235,9 @@ private static void initialize() {
         Fonts.loadFromPreferences();
         App.setAboutPanel(AboutPanel.class);
         registerFileTypes(new GCSFileProxyCreator());
+
+        // Increase TooTip time so the user has time to read the skill modifiers
+        ToolTipManager.sharedInstance().setDismissDelay(60000);
```

Comment:
This is fine for the moment, but it really should be a preference the user can change